v4l2/v4l2_decode.cpp

```diff
@@ -954,7 +954,7 @@ int32_t V4l2Decoder::onQueueBuffer(v4l2_buffer* buf)
     CHECK(buf);
     uint32_t type = buf->type;
     CHECK(type == V4L2_BUF_TYPE_VIDEO_CAPTURE_MPLANE
-        || V4L2_BUF_TYPE_VIDEO_OUTPUT_MPLANE);
+        || (unsigned int)V4L2_BUF_TYPE_VIDEO_OUTPUT_MPLANE);
```

Comment:
is this check has logical error?
should be 

CHECK(type == V4L2_BUF_TYPE_VIDEO_CAPTURE_MPLANE
-        || type == V4L2_BUF_TYPE_VIDEO_OUTPUT_MPLANE));